src/Oro/Bundle/DistributionBundle/OroKernel.php

```diff
@@ -121,7 +121,7 @@ function (\SplFileInfo $current) use (&$paths) {
     /**
      * @return array
      */
-    protected function collectBundles()
+    public function collectBundles()
```

Comment:
Instead of making this method public (that is not really good), please, move search and collection logic to separate class, and reuse it in both cases without making method collectBundles() public. Class can be called e.g. Oro\Bundle\DistributionBundle\Bundle\Collector.